package.json

```diff
@@ -121,4 +121,4 @@
     "graphql-relay": "^0.6.0",
     "lodash": "^4.17.15"
   }
-}
+}
```

Comment:
Can you put the new line back?

Reply:
Thanks! Yes, I've just committed the change to flatten my commits and remove this extra line.